docs/docusaurus.config.js

```diff
@@ -108,6 +108,14 @@ const config = {
                 label: 'Imprint',
                 href: 'https://www.askui.com/imprint',
               },
+              {
```

Comment:
Do we also need to add the script for google tag manager?

Reply:
Google recommends it as it would not work without it when a user has JavaScript disabled.